src/doip_lua_script.cpp

```diff
@@ -0,0 +1,172 @@
+#include "doip_lua_script.h"
+
+/**
+ * Default Constructor if no lua file was found.
+ */
+DoipLuaScript::DoipLuaScript(){ //if there is no config for the doip server or the ecus this default configuration will be used
+    
+    std::cout<<"Setting Default Configuration for the DoIPServer"<<std::endl;
+    A_DoIP_Announce_Num = 3;
+    A_DoIP_Announce_Interval = 500;
+    vin_ = "00000000000000000";
```

Comment:
What does the trailing underscore mean?

Reply:
nothing specific, it just should seperate the namings of some variables used in this function. I will change that